smart-contracts/test/fixtures/deploy.js

```diff
@@ -5,9 +5,9 @@ module.exports = async () => {
   // when running a mainnet fork
   if (process.env.RUN_FORK) {
     // copy .oppenzeppelin mainnet network manifest
-    copySync('.openzeppelin/mainnet.json', '.openzeppelin/unknown-31337.json')
+    // copySync('.openzeppelin/mainnet.json', '.openzeppelin/unknown-31337.json')
```

Comment:
??

Reply:
There is no these files in project so getting error.

Reply:
This file is only needed when running tests for mainnet upgrades. 

Please remove the `RUN_FORK` flag from the main coverage node task so that won't be executed. 
fwiw the manifest file is now located in the `governance/.openzeppelin` folder.